app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -110,16 +110,14 @@
   </div>
 
   <!-- drug stock -->
-  <% if current_facility_group.region.feature_enabled?(:drug_stocks) %>
-    <% unless current_facility.community? %>
-      <div class="card">
-          <h3>Drug stock report</h3>
-          <p style="text-align: left; padding: 6px 0;">Please submit your monthly hypertension drug stock report for end of <%= @for_end_of_month.to_s(:mon_year) %></p>
-          <%= link_to "Submit Drug Stock", new_webview_drug_stock_url(user_id: current_user.id,
-            access_token: current_user.access_token,
-            facility_id: current_facility.id), class: "button" %>
-      </div>
-    <% end %>
+    <% if current_facility_group.region.feature_enabled?(:drug_stocks) %>
```

Comment:
👾

Reply:
can't find, what is it? 
https://media.giphy.com/media/XdDsb6psTES0eQo8Vg/giphy.gif

Reply:
Sorry, my bad 😅  I think this line is indented one level too deep.